ansible/roles/zabbixdata_sync/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+- name: get zabbix info
+  zbx_host:
+    zbx_server: "{{ zabbix_url }}"
+    zbx_user: "{{ zabbix_user }}"
+    zbx_password: "{{ zabbix_pass }}"
+    hostgroup_names: "engint"
+    state: list
+  register: z_hosts
+  delegate_to: localhost
+
+- name: Get a list of hosts from zabbix in the zabbix hostgroup
+  zbx_hostgroup:
+    zbx_server: "{{ zabbix_url }}"
+    zbx_user: "{{ zabbix_user }}"
+    zbx_password: "{{ zabbix_pass }}"
+    name: "{{ oo_clusterid }}"
+    state: list
+    show_hosts: true
+  register: zabbix_cluster_hostgroup
+  run_once: True
+  ignore_errors: True
+  delegate_to: localhost
+
+
+- debug: var=zabbix_cluster_hostgroup.results[0].hosts|length
+- set_fact:
+   host_numberzabbix: "{{ zabbix_cluster_hostgroup.results[0].hosts|length }}"
+
+- name: "copy file on /etc/openshift_tools"
+  template:
+    src: zabbix_info.j2
```

Comment:
files should be `zabbix_data_sync.py.j2` and `zabbix_data_sync.py`
this role should be `zabbix_data_sync` instead of `zabbixdata_sync`